packages/react-server-examples/hello-world/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "react-server-examples-helloworld",
+	"private": true,
+  "version": "0.0.1",
+  "description": "",
+  "main": "HelloWorld.js",
+  "scripts": {
+		"start": "react-server-cli",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Sasha Aickin",
+  "license": "Apache-2.0",
+  "dependencies": {
```

Comment:
Eek. Thanks, @RobKopp!

Should be fixed in e686258, and I filed bug #95 to look into the root cause for the error swallowing.